Library/Homebrew/cmd/pull.rb

```diff
@@ -190,6 +190,10 @@ def pull
           safe_system 'brew', install, '--debug', f.name
         end
       end
+
+      unless ARGV.include? "--no-repair"
```

Comment:
What's the best way to check for the ` tap_name ` here? I see some logic relating to that further up; Can we recycle that logic here or will something as simple ` safe_system "brew", "tap", "--repair" if tap_name ` work?